lib/actions/addLinkToProjectAction.js

```diff
@@ -26,9 +26,9 @@ class AddLinkToProjectAction {
 
   async execute(params, context) {
     const paramsObj = this.parseParams(params);
-    const project = context;
     // eslint-disable-next-line new-cap
-    const projectLink = new this.projectLink(project, paramsObj.label, paramsObj.link);
+    const projectLink = new this.projectLink(context.project, paramsObj.label, paramsObj.link);
```

Comment:
This is great - I thought we actually made the change to ensure `project` is a property on `context` already. Keep this structure of `context.project`.